static/stylesheets/styles.css

```diff
@@ -1,479 +0,0 @@
-/**
- * Copyright 2015 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
that's weird. I checked that it was the wrong one... any way we should have only one css.